### PR TITLE
Omnibar: React to Bottom Bar feature flag changes

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL: https://app.asana.com/0/1174433894299346/1208667044987440/f

### Description
Ensure that the Omnibar code doesn’t hide the Omnibar view when the privacy config flag changes.

### Steps to test this PR

_From develop_
- [x] Install the app and navigate to a site
- [x] Open Feature Flag inventory
- [x] Disable the refactor flag
- [x] Navigate back to browser
- [x] Verify omnibar has disappeared
- [x] Open a New Tab
- [x] Verify omnibar looks good

_From this branch_
- [x] Install the app and navigate to a site
- [x] Open Feature Flag inventory
- [x] Disable the refactor flag
- [x] Navigate back to browser
- [x] Verify omnibar still there 
- [x] Open a New Tab
- [x] Verify omnibar looks good

### CODE REVIEW

**Comment by malmstein** [comment]
* **#5220** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5220?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @malmstein and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/src/main/java/com/duckduckgo/app/browser/omnibar/Omnibar.kt [hunk 1]
```diff
@@ -82,6 +82,61 @@ class Omnibar(
...
+                    // remove the default top abb bar behavior
+                    removeAppBarBehavior(binding.autoCompleteSuggestionsList)
+                    removeAppBarBehavior(binding.browserLayout)
+                    removeAppBarBehavior(binding.focusedView)
+                    removeAppBarBehavior(binding.includeNewBrowserTab.newTabLayout)
+
+                    // prevent the touch event leaking to the webView below
+                    binding.newOmnibarBottom.setOnTouchListener { _, _ -> true }
+
+                    binding.newOmnibarBottom
```
Comment (0nko) [inline app/src/main/java/com/duckduckgo/app/browser/omnibar/Omnibar.kt:108]: This shouldn’t be here. That’s what’s causing the padding issue in the NTP.

```suggestion
```

**Review by 0nko — CHANGES_REQUESTED** [review]
This solves the original problem but there is some code that needs to be removed.

I also noticed another problem: the toolbar icon is stuck on search when you disable the flag while on NTP.

**Review by 0nko — APPROVED** [review]
Nice work! :shipit:

**Pull request merged**